xplique/attributions/global_sensitivity_analysis/kernels.py

```diff
@@ -0,0 +1,113 @@
+"""
+Kernel functions for HSIC estimators.
+"""
+# pylint: disable=C0103
+
+from enum import Enum
+import tensorflow as tf
+
+
+@tf.function
+def rbf(X, Y, width):
+    """
+    Radial Basis Function kernel.
+
+    Parameters
+    ----------
+    X
+        Samples of input variable
+    Y
+        Samples of output variable
+
+    Returns
+    -------
+    Kernel matrix
+    """
+    XY = (X - Y) ** 2
+    K = tf.exp(-XY / 2 / (width**2))
```

Comment:
I suggest : `K = tf.exp(-XY / (2 * width**2))`, it should be easier to read.

Reply:
done